src/app/interfaces/article.ts

```diff
@@ -19,3 +19,10 @@ export interface Article {
   body: string; /* HTML body */
   meta?: ArticleMeta
 }
+
+export const INVALID_ARTICLE: Article = {
```

Comment:
I don't like having a fake article like this. Let the method return `null`.

Reply:
To be clear this wasn't a "fake" article it would a de-facto "error" article instead of passing null. 

I think a better approach would just to redirect the user to a 404 page or similar so it's prettier than a rendered page with an article with error states.